src/vcf/record.cpp

```diff
@@ -382,27 +378,29 @@ namespace ebi
         }
     }
 
-    void Record::check_samples_alleles(std::vector<std::string> const & alleles) const
+    void Record::check_sample_alleles(std::vector<std::string> subfields) const
     {
+        std::vector<std::string> alleles;
```

Comment:
Moved this line because `alleles` was needed for this particular method only and not anywhere else in `check_sample` method